packages/suite/src/components/suite/DropZone/index.tsx

```diff
@@ -83,13 +88,14 @@ export const useDropZone = ({ accept, onSuccess }: Props) => {
     const getWrapperProps = useMemo(
         () => () => ({
             onClick,
-            onDragEnter: prevent,
+            onDragEnter,
             onDragOver: prevent,
-            onDragLeave: prevent,
+            onDragLeave,
             onDrop,
             ref: wrapperRef,
+            className,
         }),
-        [onClick, prevent, onDrop],
+        [onClick, prevent, onDrop, className],
```

Comment:
```suggestion
        [onClick, prevent, onDrop, className, onDragEnter, onDragLeave],
```

Reply:
Fixed in [`4bb20b73`](https://github.com/trezor/trezor-suite/commit/4bb20b73d752f3325338b68921098520ad027074).